packages/react-core/src/components/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -12,7 +12,7 @@ const props = {
 };
 
 describe('TextArea', () => {
-  test('textarea input passes value and event to onChange handler', async () => {
+  test('textarea area passes value and event to onChange handler', async () => {
```

Comment:
nit - seems like you'd just say "textarea" here?

```suggestion
  test('textarea passes value and event to onChange handler', async () => {
```